MauiBundler/Plugins/Plugin.Device/Services/DeviceService.windows.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.JSInterop;
+using MauiBundler.Abstractions;
+
+namespace Plugin.Device.Services;
+
+public class DeviceService : IDeviceService
+{
+
+     [JSInvokable("installationId")]
+    public string InstallationId() =>  Preferences.Get(Constants.kInstallationGuid, null) ?? "N/A";
```

Comment:
Can we make the result nullable? Or should we throw an error when there is nothing set?